### PR TITLE
Fix incorrect regexps and modes.

### PR DESCRIPTION
**Pull request by seagle0128**

### CODE REVIEW

**Comment by ema2159** [comment]
Could you also remove de duplicated lua mode icon? I think we both added it on separate PRs 😅.

**Comment by seagle0128** [comment]
@ema2159 Already removed the duplicated lua icons.

File: all-the-icons.el [hunk 1]
```diff
@@ -743,12 +741,12 @@ directory contents"
          (padding (or padding "\t"))
          (icon (cond
                 ((file-symlink-p path)
-                 (all-the-icons-octicon "file-symlink-directory" :height 1.0))
+                 (all-the-icons-octicon "file-symlink-directory" :height 1.0 :v-adjust 0.01))
```
Comment (wyuenho) [inline all-the-icons.el]: Can you remove these directory alignment changes from this PR please?

**Comment by seagle0128** [comment]
Close it due to the mixed branch issue. New PR is #176 .

**Pull request closed without merging**